angular/projects/admin-nrpti/src/app/mines/mines-rows/mines-table-row.component.html

```diff
@@ -1,8 +1,25 @@
 <td scope="row" data-label="Name" class="col-3">{{ rowData.name || '-' }}</td>
+<td scope="row" data-label="Permit Numbers" class="col-2">
+  <ul *ngIf="rowData.permitNumbers.length > 0; else elseBlock">
+    <li *ngFor="let permit of rowData.permitNumbers">{{ permit }}</li>
+  </ul>
+  <ng-template #elseBlock>-</ng-template>
+</td>
+<td scope="row" data-label="Type" class="col-1">{{ rowData.type || '-' }}</td>
 <td scope="row" data-label="Permittee" class="col-2">{{ rowData.permittee || '-' }}</td>
-<td scope="row" data-label="Type" class="col-2">{{ rowData.type || '-' }}</td>
-<td scope="row" data-label="Status" class="col-2">{{ rowData.status || '-' }}</td>
-<td scope="row" data-label="Region" class="col-2">{{ rowData.region || '-' }}</td>
+<td scope="row" data-label="Region" class="col-1">{{ rowData.region && displayRegion(rowData.region) || '-' }}</td>
+
+<td scope="row" data-label="Published State" class="col-2 d-flex">
+  <div *ngIf="rowData.publishedDate; then published else unpublished"></div>
+  <ng-template #published class="test">
+    <i class="material-icons published-green">check_circle</i>
+    <span class="publish-label pl-1 published-green">Published</span>
+  </ng-template>
+  <ng-template #unpublished class="test">
+    <i class="material-icons ">cancel</i>
+    <span class="publish-label pl-1">Unpublished</span>
+  </ng-template>
+</td>
```

Comment:
We should probably base publish state off of whether 'public' exists in the read array. Published date my be populated if a mine is published and then unpublished.

Reply:
Good point. Updated to look at the read array.